napari/components/viewer_model.py

```diff
@@ -828,7 +831,9 @@ def _add_layers_with_plugins(
         """
         from ..plugins.io import read_data_with_plugins
 
-        layer_data = read_data_with_plugins(path_or_paths, plugin=plugin) or []
+        layer_data, hookimpl = (
+            read_data_with_plugins(path_or_paths, plugin=plugin) or []
```

Comment:
Does this raise 

```python
layer_data, hookimpl = []
```

`ValueError: not enough values to unpack (expected 2, got 0)` 🤔

Reply:
ahh, good point. it won't actually, because i modified `read_data_with_plugins` to always return a tuple, but forgot to update the `or []` here (which will now never hit)

Reply:
👍🏼

Reply:
Coolio, wanna update? =)